### PR TITLE
Fix/ocp4.11

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

Make TS work on OCP 4.11

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by fedinskiy** [comment]
run tests

**Comment by jsmrcka** [comment]
run tests

**Comment by fedinskiy** [comment]
run tests

**Review by mjurc — APPROVED** [review]
LGTM, thank you for the contribution.

File: sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/openshift/OpenShiftHibernateReactiveMsSQLIT.java [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 import io.quarkus.test.scenarios.OpenShiftScenario;
 import io.quarkus.ts.hibernate.reactive.MsSQLDatabaseHibernateReactiveIT;
+import org.junit.jupiter.api.Disabled;
```
Comment (mjurc) [inline sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/openshift/OpenShiftHibernateReactiveMsSQLIT.java:5]: Actually, bad sorting on imports here, see the CI failure.

**Review by mjurc — CHANGES_REQUESTED** [review]

**Review by mjurc — APPROVED** [review]

**Comment by fedinskiy** [comment]
`cannot locate openjdk-11` ? Sounds weird. I will rerun OCP tests

**Comment by fedinskiy** [comment]
run tests

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Why is the PR targeting 2.7, we should start with main and bulk-backport the necessary fixes to 2.7? cc @pjgg 

I'm not excited about dropping MSSQL coverage. There seems to be workaround with setting security context. We need to see if we can have this done in FW and how much work is it.

**Review by rsvoboda — APPROVED** [review]
There is workaround for MSSQL image trouble, but we need to investigate if that is safe (doesn't need higher privileges) and we can do MSSQL specific adjustment in FW. There is dedicated Trello card for that.

Plan for now is to disable MSSQL OCP tests, Fedor will create similar PR for `main`

**Pull request merged**